File: app/src/main/java/org/mifos/mobile/ui/views/SavingsMakeTransferMvpView.java
package org.mifos.mobile.ui.views;

import org.mifos.mobile.models.templates.account.AccountOptionsTemplate;
import org.mifos.mobile.ui.views.base.MVPView;

/**
 * Created by Rajan Maurya on 10/03/17.
 */

public interface SavingsMakeTransferMvpView extends MVPView {

    void showUserInterface();

    void showSavingsAccountTemplate(AccountOptionsTemplate accountOptionsTemplate);

    void showToaster(String message);

    void showError(String message);

    void showProgressDialog();

    void hideProgressDialog();
}

File: app/src/main/java/org/mifos/mobile/ui/views/SavingsMakeTransferMvpView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.models.templates.account.AccountOptionsTemplate
import org.mifos.mobile.ui.views.base.MVPView

/**
 * Created by Rajan Maurya on 10/03/17.
 */
interface SavingsMakeTransferMvpView : MVPView {
    fun showUserInterface()
    fun showSavingsAccountTemplate(accountOptionsTemplate: AccountOptionsTemplate?)
    fun showToaster(message: String?)
    fun showError(message: String?)
    fun showProgressDialog()
    fun hideProgressDialog()
}